crypto/src/proofs/transparent.rs

```diff
@@ -652,20 +606,17 @@ mod tests {
         let rsk = sk_sender.spend_auth_key().randomize(&spend_auth_randomizer);
         let nk = *sk_sender.nullifier_key();
         let ak = sk_sender.spend_auth_key().into();
-        let mut nct = merkle::BridgeTree::<note::Commitment, 32>::new(5);
-        nct.append(&note_commitment);
-        let anchor = nct.root2();
-        nct.witness();
-        let merkle_path = nct.authentication_path(&note_commitment).unwrap();
+        let mut nct = penumbra_tct::Eternity::new();
+        nct.insert(penumbra_tct::Keep, note_commitment).unwrap();
```

Comment:
Rather than re-exporting the inner `Witness::Keep` and `Witness::Forget` variants from the crate root, then calling them `tct::Keep`, it would be clearer if the calling code just imported `Witness` and wrote
```
nct.insert(Witness::Keep, note_commitment);
```